packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -170,7 +174,19 @@ export class JobService {
         ),
         gt_url: generateBucketUrl(dto.groundTruth, requestType),
       },
-      job_bounty: await this.calculateJobBounty(elementsCount, tokenFundAmount),
+      job_bounty:
+        requestType === JobRequestType.IMAGE_SKELETONS_FROM_BOXES
+          ? await this.calculateJobBounty(
```

Comment:
This could be refactored to pass an objtect to calculateJobBounty instead of 4 params. Also you do not need this conditional call